src/test/java/com/bringframework/reader/DefaultBeanDefinitionReaderTest.java

```diff
@@ -32,7 +32,7 @@ class DefaultBeanDefinitionReaderTest {
 
   @Test
   void shouldRegisterBeans() {
-    underTest.registerBeans("com.bringframework");
+    underTest.registerBeans("com.bringframework.reader");
```

Comment:
Was it change required in this PR? If not, then please revert it
btw, I saw that @olehhaliak did renaming of BeanReader to BeanScanner, so potentially here you would have unnecessary conflict

Reply:
It's required, cause tests were failing